CHANGELOG.asciidoc

```diff
@@ -3,6 +3,236 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.12.0]]
+=== Beats version 7.12.0
+https://github.com/elastic/beats/compare/v7.11.2...v7.12.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Fix panic with inline SSL when the certificate or key were small than 256 bytes. {pull}23820[23820]
+
+*Auditbeat*
+
+*Filebeat*
+
+- Add fileset to ingest PostgreSQL CSV logs. {pull}23334[23334]
```

Comment:
@jsoriano  is this a breaking change or shall we move to fix?

Reply:
This should be in "Added", thanks!